src/main/java/uk/gov/pay/api/ledger/service/TransactionSearchService.java

```diff
@@ -2,6 +2,7 @@
 
 import com.google.common.collect.ImmutableSet;
 import com.google.inject.Inject;
+import org.skife.jdbi.v2.sqlobject.Transaction;
```

Comment:
don't think this is what you wanted?

Reply:
oops, alt + enter too fast (sometime IntelliJ is not smart enough).